File: blockchain-info-aggregation-stream-parent/blockchain-info-aggregation-stream-domain/src/main/java/info/blockchain/aggregation/domain/Output.java
package info.blockchain.aggregation.domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Value;

@JsonIgnoreProperties({"spent", "tx_index", "type", "addr", "n", "script"})
@Value
public class Output {

    @JsonProperty("value")
    public final long value;

}

File: blockchain-info-aggregation-stream-parent/blockchain-info-aggregation-stream-domain/src/main/java/info/blockchain/aggregation/domain/Output.kt
package info.blockchain.aggregation.domain

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties("spent", "tx_index", "type", "addr", "n", "script")
class Output(@JsonProperty("value") val value: Long = 0) {
}
